### PR TITLE
Unpin flake

### PR DESCRIPTION
**Pull request by beliaev-maksim**
change resolver
update flake8

### CODE REVIEW

**Comment by beliaev-maksim** [comment]
@DnPlas 
there are no updates in .txt file

File: charms/kserve-controller/requirements-integration.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 aiohttp
 jinja2
-juju==3.0.4
+juju<3.0
```
Comment (DnPlas) [inline charms/kserve-controller/requirements-integration.in:3]: Please add a comment that states why we are pinning to this version.

**Review by DnPlas — COMMENTED** [review]
Small comment.
Also, for future PRs please make sure the title and description match the changes you are proposing.

File: charms/kserve-controller/requirements-integration.in (hunk 1 above)
Comment (beliaev-maksim) [reply charms/kserve-controller/requirements-integration.in:3]: done

**Review by DnPlas — APPROVED** [review]

**Pull request merged**